CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 * Venmo
   * Add `setIsFinalAmount()` to `VenmoRequest`
+* ThreeDSecure
+  * Call cleanup method to resolve `Cardinal.getInstance` memory leak (fixes #898)
```

Comment:
Does this fix 898? I think that is the memory leak that is still open with Cardinal, but I could be wrong

Reply:
Ah you may be right, this only fixes _half_ of the memory leak, so maybe not worth calling out until we get a new version from Cardinal that fully fixes it. What do you think?